packages/tendermint-rpc/src/index.ts

```diff
@@ -78,6 +47,38 @@ export {
   Version,
   Vote,
   VoteType,
-} from "./responses";
+  AbciInfoRequest,
+  AbciQueryParams,
+  AbciQueryRequest,
+  BlockRequest,
+  BlockchainRequest,
+  BlockResultsRequest,
+  BroadcastTxRequest,
+  BroadcastTxParams,
+  CommitRequest,
+  GenesisRequest,
+  HealthRequest,
+  Method,
+  Request,
+  QueryTag,
+  StatusRequest,
+  SubscriptionEventType,
+  TxParams,
+  TxRequest,
+  TxSearchParams,
+  TxSearchRequest,
+  ValidatorsRequest,
+  ValidatorsParams,
+} from "./legacy";
```

Comment:
Why not just `export * from "./legacy"`?

Reply:
I'd rather see that is exported in the top-level file, especially since wee need to take care of collisions now. I was thinking about `export * as legacy from "./legacy"` but this would be heavily breaking.